### DISCUSSION TITLE
The difference between aWETH and aETH (Aave interest tokens)

### DISCUSSION THREAD

**Discussion by Verde888** [smartcontractkit/full-blockchain-solidity-course-py#839] [Q&A]
Out of curiosity...
It seems that there are two tokens and I cant figure out the difference.
When I deposit WETH on Aave, I receive aETH on my MetaMask wallet.
@PatrickAlphaC seems to receive aWETH.

Any ideas? :))

Contracts:
aWETH: 0x030ba81f1c18d280636f32af80b9aad02cf0854e (???)
aETH: 0x3a3A65aAb0dd2A17E3F1947bA16138cd37d08c04

**Comment by cromewar** [comment]
Hello @Verde888 Weth goes for Wrapped Eth which is just something devs use in order to be able to exchange Weth with other tokens, However Aave is constantly changing and aEth and aWeth so I wont worry much about it.

**Reply by Verde888** [reply]
Hey @cromewar! Yes, this is all true, I was just wondering if anyone had some more specific info, e.g. eWETH has been replaced by aETH in the last two months or something...

**Reply by cromewar** [reply]
I don't think so, the contract is still aWeth, not sure why aave is giving aEth instead of aWeth. I'll do more research on this.

**Reply by cromewar** [reply]
Could find anything relevant neither on the forums neither on aave docs, did the lending pool gave the option to choose which token you were going to receive?

**Reply by Verde888** [reply]
Hey, I did it only in Brownie, following the course. I realized that I received different tokens than Patrick.
I promise to write here if I learn the difference one day ;))
